src/api/goods.ts

```diff
@@ -10,10 +10,18 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods
+      .sort((goodA: Good, goodB: Good) => goodA.name.localeCompare(goodB.name))
+      .slice(0, 5))
+    .catch(error => {
```

Comment:
If you add error processing, then it must be carried out in a separate ```if ``` with ```return``` before processing the received data